macaroni-os/kit-fixups
# =www-client/firefox-131.0 and =mail-client/thunderbird-131.0

Summary
========
* updated to 131.0

Test Plan
========
* Doit
```
 ╰ $ doit
WARNING  dict key unmasked overwritten.                                                                                                                                                                                                                                                                             
INFO     Autogen: www-client/firefox-131.0                                                                                                                                                                                                                                                                          
INFO     Download from https://dev.gentoo.org/~juippis/mozilla/patchsets/firefox-131-patches-01.tar.xz verified as valid tar.xz archive.                                                                                                                                                                            
INFO     Download from https://archive.mozilla.org/pub/firefox/releases/131.0/source/firefox-131.0.source.tar.xz verified as valid tar.xz archive.                                                                                                                                                                  
INFO     Created: firefox-131.0.ebuild 
```
* Make it available in a own repo tree or by seeding into a local overlay
* Emerge it
```
~ # emerge www-client/firefox

These are the packages that would be merged, in order:

Calculating dependencies... done!
[ebuild     U  ] www-client/firefox-131.0 [130.0]

Would you like to merge these packages? [Yes/No] 
>>> Verifying ebuild manifests
>>> Running pre-merge checks for www-client/firefox-131.0
 * Checking for at least 6600 MiB disk space at "/run/portage/www-client/firefox-131.0/temp" ...                                                                                                                                                                                   [ ok ]
>>> Emerging (1 of 1) www-client/firefox-131.0::public-overlay
>>> Installing (1 of 1) www-client/firefox-131.0::public-overlay
>>> Jobs: 1 of 1 complete                           Load avg: 12.7, 32.4, 20.0

 * Messages for package www-client/firefox-131.0:

 * 
 * Unfortunately Firefox-100.0 breaks compatibility with some sites using 
 * useragent checks. To temporarily fix this, enter about:config and modify 
 * network.http.useragent.forceVersion preference to "99".
 * Or install an addon to change your useragent.
 * See: https://support.mozilla.org/en-US/kb/difficulties-opening-or-using-website-firefox-100
 * 
>>> Auto-cleaning packages...

>>> No outdated packages were found on your system.

 * GNU info directory index is up-to-date.

```

```
~ # emerge -av mail-client/thunderbird

These are the packages that would be merged, in order:

Calculating dependencies... done!
[ebuild     U  ] mail-client/thunderbird-131.0::public-overlay [130.0::net-kit] USE="X clang dbus openh264 pulseaudio system-harfbuzz system-jpeg system-libevent -debug -eme-free -hardened -hwaccel -jack -lto -pgo (-selinux) -sndio -system-av1 -system-icu -system-libvpx -system-png -system-webp -wayland -wifi" L10N="de -af -ar -ast -be -bg -br -ca -cak -cs -cy -da -dsb -el -en-CA -en-GB -es-AR -es-ES -et -eu -fi -fr -fy -gd -gl -he -hr -hsb -hu -id -is -it -ja -ka -kab -kk -ko -lt -lv -ms -nb -nl -nn -pa -pl -pt-BR -pt-PT -rm -ro -ru -sk -sl -sq -sr -sv -th -tr -uk -uz -vi -zh-CN -zh-TW" 0 KiB

Total: 1 package (1 upgrade), Size of downloads: 0 KiB

Would you like to merge these packages? [Yes/No] 
>>> Verifying ebuild manifests
>>> Running pre-merge checks for mail-client/thunderbird-131.0
 * Checking for at least 6400 MiB disk space at "/run/portage/mail-client/thunderbird-131.0/temp" ...                                                                                                                                                                              [ ok ]
>>> Emerging (1 of 1) mail-client/thunderbird-131.0::public-overlay
>>> Installing (1 of 1) mail-client/thunderbird-131.0::public-overlay
>>> Jobs: 1 of 1 complete                           Load avg: 9.0, 29.8, 25.3

 * Messages for package mail-client/thunderbird-131.0:

 * Optional runtime features:
 *   net-libs/libotr for encrypted chat support
>>> Auto-cleaning packages...

>>> No outdated packages were found on your system.

 * GNU info directory index is up-to-date.

```
* Running it
```
 ╰ $ firefox -version
Mozilla Firefox 131.0
```
```
 ╰ $ thunderbird -version
Mozilla Thunderbird 131.0
```

![image](https://github.com/user-attachments/assets/f65825d3-96ee-43cc-9856-bd428f3b4f98)
![image](https://github.com/user-attachments/assets/9cf1f4d1-20fd-4343-8e5a-6133232ed81c)
